app/views/omni_nav_v2/_off_canvas_nav.html.erb

```diff
@@ -23,6 +23,7 @@
       <div id="main-logo" class="toggle-logo">
         <a class="default off-logo" href="/" title="Chapman University Website Home Page">
           <!-- logo set as background image by class -->
+          Chapman University
```

Comment:
I updated these 2 views because they were showing WAVE errors, even though the real velocity formats in Cascade have already been updated to fix them. The corresponding views we use to mimic those formats were not updated, and it's kind of distracting when I'm trying to look for real errors.